File: app/src/main/java/com/example/kirill/retrofittry/Interfaces/GetQuestions.java
package com.example.kirill.retrofittry.Interfaces;

import com.example.kirill.retrofittry.Parsers.Question;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;


public interface GetQuestions {
    @GET("questions")
    Call<GetThemes.Questions> all();

    @GET("questions/{id}")
    Call<Question> select(@Path("id") int id);
}

File: app/src/main/java/com/example/kirill/retrofitTry/interfaces/GetQuestions.kt
package com.example.kirill.retrofitTry.interfaces

import com.example.kirill.retrofitTry.Parsers.Question
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface GetQuestions {
    @GET("questions")
    fun all(): Call<GetThemes.Questions>

    @GET("questions/{id}")
    fun select(@Path("id") id: Int): Call<Question>
}